plugins/lando/README.md

```diff
@@ -0,0 +1,30 @@
+
+# Lando ZSH (lando-zsh)
+This plugin adds aliases for using various languages and frameworks with [Lando](https://docs.lando.dev/basics/) for Docker. It will only run within lando-driven project directories.
+
+#### USAGE:
+```
+plugins=(... lando)
+```
+
+#### ALIASES:
+| Alias | Description |
+|:-:|:-:|
+| `artisan`  | `lando artisan`  |
+| `composer`  | `lando composer`  |
+| `drush`  | `lando drush`  |
+| `gulp`  | `lando gulp`  |
+| `npm`  | `lando npm`  |
+| `wp`  | `lando wp`  |
+| `yarn`  | `lando yarn`  |
+
+#### How It Works:
+This plugin removes the requirement to type `lando` before a command. It prepends 'lando' for commands executed in the same directory as a .lando.yml config file, but preserves the use of non-lando commands with the same name.
```

Comment:
This explanation isn't accurate. It will also use lando if a lando.yml file is found in a parent directory, always inside `$LANDO_ZSH_SITES_DIRECTORY`.